servers/kamyshowdown/init.c

```diff
@@ -61,6 +61,9 @@ class CustomMission: MissionServer
 		return m_player;
 	}
 
+	// 0 or less is always false
```

Comment:
So I've updated the logic in a fix to the feature branch for the decide function. Will need to edit this file accordingly.

Reply:
@jsirianni Actually we will just need to remove these comments. I updated the function to be a little more clear in the other PR

Reply:
For clarity it's just lines 64, 65, & 66